jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -5354,4 +5354,42 @@ public IntStream codePoints() {
 		return value;
 	}
 /*[ENDIF] Java10 */	
+
+/*[IF Java11]*/
+	/**
+	 * Returns a string object containing the character (Unicode code point)
+	 * specified.
+	 * 
+	 * @param codePoint
+	 *            a Unicode code point.
+	 * @return a string containing the character (Unicode code point) supplied.
+	 * @throws IllegalArgumentException
+	 *             if the codePoint is not a valid Unicode code point.
+	 * @since 11
+	 */
+	static String valueOfCodePoint(int codePoint) {
+		String string;
+		// Save less often check (codePoint > Character.MAX_CODE_POINT) last
```

Comment:
Comment is no longer accurate

Reply:
The comment in question has been removed.